decidim-core/app/models/decidim/user.rb

```diff
@@ -89,6 +90,21 @@ def follows?(followable)
       Decidim::Follow.where(user: self, followable: followable).any?
     end
 
+    def following
```

Comment:
Docs

Reply:
Can't you use a polymorphic `through` association for this?

Reply:
Nope, Rails does not allow it 😞